src/styles/admin-core/_content-pages-preview.scss

```diff
@@ -264,6 +264,7 @@ $l-container-padding-xs: 2rem;
 		&:focus,
 		.focus-visible:focus:not(:focus-visible) {
 			outline: 2px solid $teal !important;
+			margin: 4px 0;
```

Comment:
doesn't this cause a jump in the page when you focus the button?
if it does, maybe its better to put the margin on the button from the start?

Reply:
Good call